### DISCUSSION TITLE
Can we read the dataset by the folder names without annotation text

### DISCUSSION THREAD

**Discussion by lxrswdd** [open-mmlab/mmclassification#1601] [General]
Hi, we have to generate an `ann_file` in the `.josn` file.
Can MMpretrain reads the data and label it by the folder name? Just like `torchvision.datasets.ImageFolder`.

**Comment by Ezra-Yu** [comment]
Yes, refer to https://mmpretrain.readthedocs.io/en/latest/user_guides/dataset_prepare.html#subfolder-format.
